src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt

```diff
@@ -94,6 +94,8 @@ abstract class RsFunctionImplMixin : RsStubbedNamedElementImpl<RsFunctionStub>,
     final override val innerAttrList: List<RsInnerAttr>
         get() = block?.innerAttrList.orEmpty()
 
+    override fun getNameIdentifier(): PsiElement? = findLastChildByType(RsElementTypes.IDENTIFIER)
```

Comment:
Hmm, looks like it isn't needed for async feature. Is it tested?

Reply:
True, this is no longer needed and was left from the previous implementation. Removed.